src/angular/partials/search.html

```diff
@@ -1,37 +1,38 @@
-      <div class="row">
-        <h1>UNHCR Services Advisor</h1>
-        <button id="mapButton" type="button" class="btn btn-default">
-          <span class="glyphicon glyphicon-map-marker" aria-hidden="true"></span> Map
-        </button>
-        <h2>Heading 2</h2>
-        <h3>Heading 3</h3>
-        <p>
-          To find a service, select from the category or region below. Use the filter button above to further refine your search.
-        </p>
-      </div>
-      <div class="row">
-        <h4>Category</h4>
-        <div class="list-group">
-          <div ng-repeat="category in categories">
-            <a class="list-group-item">
-                {{category.name}} <span class="badge">{{category.count}}</span>
-            </a>
-            <div id="{{category.name}}Activities" class="activities">
-              <a ng-repeat="activity in category.activities" ng-click="selectCategory(activity.name)" class="list-group-item">
-                <span class="icon glyphicon glyphicon-star-empty"></span> {{activity.name}}
-                <span class="badge">{{activity.count}}</span>
-              </a>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="row">
-        <h4>Region</h4>
-        <!-- populating the catgories from the SearchCtrl  -->
-        <div ng-repeat="region in regions">
-          <a href="#/results?region={{region}}" class="list-group-item">
-            <span class="icon glyphicon glyphicon-star-empty"></span> {{region.name}}
-            <span class="badge">{{region.count}}</span>
-          </a>
-        </div>
+<div class="row">
```

Comment:
@cynthialee you mean because at the point where it gets injected it's already indented? That's true but it's easier to run auto-format in your IDE and have it start on the far left. Also we're going to be editing the file in the IDE and not looking at the actual outputted HTML too often (except using chrome debugger) so I think it's a fair trade-off